test/Microsoft.AspNet.Http.Extensions.Tests/HeaderDictionaryTypeExtensionsTest.cs

```diff
@@ -103,10 +103,11 @@ public void GetListT_KnownTypeWithValidValue_Success()
         public void GetListT_KnownTypeWithMissingValue_Null()
         {
             var context = new DefaultHttpContext();
-
-            var result = context.Request.GetTypedHeaders().GetList<MediaTypeHeaderValue>(HeaderNames.Accept);
-
-            Assert.Null(result);
+            var result1 = context.Request.GetTypedHeaders().GetList<MediaTypeHeaderValue>(HeaderNames.Accept);
```

Comment:
Must have been something extra that was there at some point in the commit changes; will revert